### PR TITLE
ci: use our bot to do auto-merging of Dependabot PRs

### PR DESCRIPTION
**Pull request by AlCalzone**

### CODE REVIEW

**Comment by AlCalzone** [comment]
@zwave-js-bot approve

**Comment by AlCalzone** [comment]
@zwave-js-bot approve

File: .github/bot-scripts/shouldAutomerge.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+				update.updateType === "version-update:semver-minor")) ||
+		// production dependencies: patch
+		(update.dependencyType === "direct:production" &&
+			update.updateType === "version-update:semver-patch") ||
+		// production dependencies: minor if security alert
+		(update.dependencyType === "direct:production" &&
+			update.updateType === "version-update:semver-minor" &&
+			update.alertState === "OPEN") ||
+		// indirect dependencies: minor if security alert
+		(update.dependencyType === "direct:production" &&
```
Comment (OXINARF) [inline .github/bot-scripts/shouldAutomerge.js:51]: indirect?

Comment (AlCalzone) [reply .github/bot-scripts/shouldAutomerge.js:51]: good catch

**Pull request merged**